app/src/apps/market_trends/management/commands/load_market_data.py

```diff
@@ -1,30 +1,46 @@
+import logging
+
 import pandas as pd
 import requests
 from django.core.management.base import BaseCommand
 from django.utils import timezone
 from requests.exceptions import RequestException
-from src.apps.market_trends.models import CurrencyData
+from src.apps.market_trends.models import CurrencyData, ExecutionLog
 from src.apps.market_trends.services import CurrencyDataService
 
+logger = logging.getLogger(__name__)
+MSG_WARNING = "Dias que não tem registros nos útimos 365 dias"
+
 
 class Command(BaseCommand):
     help = "Carrega dados históricos de criptomoedas e calcula médias móveis simples MMS"
 
-    def handle(self, *args, **kwargs):
+    def handle(self, *args, **kwargs):  # sourcery skip: extract-method
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `handle`

Reason for update: The function's behavior has been extended to include logging of the data loading process and errors.

Suggested new docstring:
```
Carrega dados históricos de criptomoedas, calcula médias móveis simples (MMS) e registra o processo e erros.
```